lib/Dialect/FIRRTL/Transforms/LowerXMR.cpp

```diff
@@ -47,6 +47,26 @@ using hw::InnerRefAttr;
 /// and Top.Foo.xmr_wire and Top.Zoo.Foo.xmr_wire for different instance paths
 /// that exist in the circuit.
 
+namespace {
+struct XMRNode {
+  using nextNodeOnPath = std::optional<size_t>;
```

Comment:
not 100% sure about the convention, but 
```suggestion
  using NextNodeOnPath = std::optional<size_t>;
```

Reply:
I was jut using the existing naming, just hoisted it so I can use the types :sweat_smile: .  But I think you're right re:types are capitalized, thanks!